mitodl/mit-open
# Remove some offerors

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4287

### Description (What does it do?)
- Removes 3 offerors (CSAIL, Center for Transportation & Logistics, Schwarzman)
- Updates prolearn ETL pipeline to set `offered_by=None` for affected resources
- Updates `populate_featured_lists` mgmt command and featured resources API to process all remaining offerors, not just ones identified as "featured"


### How can this be tested?
- Run `./manage.py update_offerors` followed by `./manage.py update_index --all`
- Check that the 3 offerors and their channels are gone (http://localhost:8063/api/v0/channels?channel_type=offeror, http://localhost:8063/api/v1/offerors)
- Run `./manage.py backpopulate_prolearn_data`, you should end up with some courses that have a platform of `csail`, `ctl`, or `scc` but a null offeror field.
